Automattic/pocket-casts-ios
# Playback 2024: Top 5 Podcasts List

| 📘 Part of: #2250 |
|:---:|

Completes #2317

<img src="https://github.com/user-attachments/assets/6bcba790-6cf5-4b53-8bae-409a12abe5ce" width=300>

## To test

* Enable the `endOfYear2024` feature flag
* Restart the app
* Visit Profile
* Tap the Playback 2024 Cell
* Tap through until you get to this screen
* Verify that the list is shown and accurate

## Checklist
